tests/foreman/api/test_reporttemplates.py

```diff
@@ -591,3 +591,67 @@ def test_positive_schedule_entitlements_report(setup_content, target_sat):
         )
         assert vm.hostname in data_csv
         assert DEFAULT_SUBSCRIPTION_NAME in data_csv
+
+
+@pytest.mark.no_containers
+@pytest.mark.tier3
+def test_positive_generate_job_report(setup_content, target_sat, rhel7_contenthost):
```

Comment:
Doesn't matter, since the test is just using this fixture as setup for triggering the new remote execution report template, not testing anything contenthost specific. The actual host just runs `pwd` in this case, for example